libs/model/src/aggregates/feed/GetGlobalActivity.query.ts

```diff
@@ -1,20 +1,61 @@
-import { Query } from '@hicommonwealth/core';
+import { cache, CacheNamespaces, Query } from '@hicommonwealth/core';
 import * as schemas from '@hicommonwealth/schemas';
-import { getUserActivityFeed } from '../../utils/getUserActivityFeed';
+import { QueryTypes } from 'sequelize';
+import { z } from 'zod';
+import { models } from '../../database';
+import { baseActivityQuery } from '../../utils/getUserActivityFeed';
 
-export function GetGlobalActivity(): Query<typeof schemas.ActivityFeed> {
+export function GetGlobalActivity(): Query<typeof schemas.GlobalFeed> {
   return {
-    ...schemas.ActivityFeed,
+    ...schemas.GlobalFeed,
     auth: [],
     secure: false,
     body: async ({ payload }) => {
       const { comment_limit = 3, limit = 10, cursor = 1 } = payload;
-      return await getUserActivityFeed({
-        user_id: 0,
-        comment_limit,
-        limit,
-        cursor,
+
+      const rankedThreadIds = await cache().sliceSortedSet(
```

Comment:
Similar to comment above, caching layer surrounding commands and queries (middleware)